progs/get-subseqs/main.go

```diff
@@ -30,5 +56,43 @@ func main() {
 		defer coordFile.Close()
 	}
 
-	fmt.Println("I should be processing the coordinate stream now!")
+	//######################
+
+	output := ""
+	fileScannerCoord := bufio.NewScanner(coordFile)
+
+	for fileScannerCoord.Scan() {
+		lineCoord := fileScannerCoord.Text()
+		if len(lineCoord) == 0 {
+			continue
+		}
+		split_str := strings.Fields(lineCoord)
+		if len(split_str) != 3 {
+			fmt.Println("Error, length of coordinates")
+		}
+		for i, v := range split_str {
+			split_str[i] = strings.TrimSpace(v)
+		}
+
+		for _, v := range sequences {
```

Comment:
So we are doing a linear search here. Nothing wrong with that if that is the best we could come up with, and if the number of sequences is short, but had you put the sequences into a `map` you could get a constant time lookup.